src/layer/BingTiledImageLayer.js

```diff
@@ -67,7 +67,7 @@ define([
              * @default A value of (WorldWind.OFFSET_INSET_PIXELS, 5, WorldWind.OFFSET_PIXELS, 5) provides a
              * 5px margin inset from the lower right corner of the screen.
              */
-            this.logoPlacement = new Offset(WorldWind.OFFSET_INSET_PIXELS, 5, WorldWind.OFFSET_PIXELS, 5);
+            this.logoPlacement = new Offset(WorldWind.OFFSET_INSET_PIXELS, 5, WorldWind.OFFSET_PIXELS, 20);
 
```

Comment:
Is it possible to make the y offset driven by the height of the text attributions rather than hard coded? In the default case where there are no attributions other than Bing, there is currently just blank unused space at the bottom.